### DISCUSSION TITLE
GLB model has different size between preview mode and ar mode

### DISCUSSION THREAD

**Discussion by hoangthai267** [google/model-viewer#2294] [Q&A]
### Description

The preview of my model in preview section is quite big. But when I enter AR mode, it becomes very small. Does anyone have any suggestion ?

Preview section: 
![Screenshot_20210414-165742_Chrome](https://user-images.githubusercontent.com/12732489/114692565-f04e5580-9d42-11eb-9a02-3a4466d8dc40.jpg)

AR section:
![Screenshot_20210414-161112_Chrome](https://user-images.githubusercontent.com/12732489/114692255-aa918d00-9d42-11eb-8332-dea60649fde3.jpg)

#### Live Demo

Not available for now

### Version

- model-viewer: https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
The size in AR is dictated by the model; all glTF files are in meters, so if it was exported incorrectly it may be at a strange scale. In 3D mode we automatically set it to match the model's bounds, so I believe the only way you could be so zoomed in is if you set the `camera-orbit` radius yourself.

**Reply by hoangthai267** [reply]
Apologies for the late reply, I will try to set camera-orbit in my project soon. Thanks.

**Reply by rianlukog** [reply]
Facing the same issue: https://github.com/google/model-viewer/discussions/2526#discussion-3437927

My 3D view port looks good but while activating AR the model seems to be extremely off-center and very tiny hard to find sometimes
